openlibrary/components/MergeUI/AuthorRoleTable.vue

```diff
@@ -36,7 +36,7 @@ export default {
     },
     computed: {
         fields() {
-            return _.uniq(_.flatMap(this.roles, Object.keys));
+            return _.uniq(_.flatMap(this.roles, function(input) { const keys=Object.keys(input); keys.sort; return keys; }));
```

Comment:
Did you mean `.sort()`? Why sorting here?

Reply:
Is this what you were going for?
```suggestion
            return _.uniq(_.flatMap(this.roles, Object.keys)).sort();
```